test/unit/services/validators/test-common-validator.js

```diff
@@ -602,4 +602,39 @@ describe('services/validators/common-validator', function () {
       done()
     })
   })
+
+  describe('isValidStatus', function () {
```

Comment:
there will be multiple status values but we should never allow the user to supply what is used, so don't think this is needed